db/errors.go

```diff
@@ -29,6 +29,7 @@ var KnownErrors = map[string]string{
 	"max dial attempts exceeded":                 models.NetErrorMaxDialAttemptsExceeded,
 	"host is down":                               models.NetErrorHostIsDown,
 	"stream reset":                               models.NetErrorStreamReset,
+	"stream closed":                              models.NetErrorStreamReset,
```

Comment:
should we create a new `models.NetErrorStreamClosed` entry for this error?
(I'm not 100% sure if it just means the same as the "stream reset" one)

Reply:
yeah, I'm lazy and didn't want to add a migration just for this error code 🙈 

It's not exactly the same as a stream reset but for analysis purposes I'd say we can treat it equally